### DISCUSSION TITLE
WinUtil on Win11 caused a required password change request by the organization while retaining the original password after completing the prompt

### DISCUSSION THREAD

**Discussion by 32ic** [ChrisTitusTech/winutil#2833] [General]
As the title says. Sadly I do not remeber what options I used in WinUtil on my fresh Win11 install... I remember distinctly trying the same password first and then using the same but counting up the last number from 1 to 2. After booting the next day I had to use 1 and not 2.

**Comment by 32ic** [comment]
Also I have the effect that the Windows Security Center does not remember the dismissal of reminders for security options .
![{C0EBB40D-A5EE-452D-85CA-03FED87C780A}](https://github.com/user-attachments/assets/c827713b-f26a-483c-a59b-b86426ac7131)

I now actually found out I can "Get Installed" so here is what I used.
![{47C33C73-CC9F-4EE4-A1D4-93D1869C8F42}](https://github.com/user-attachments/assets/5c9fef2d-4501-4492-a151-c311ec9301b8)

I dont believe having used any of the "Config"-tab options

**Comment by ChrisTitusTech** [comment]
Removed all password requirements
